README.md

```diff
@@ -50,6 +50,21 @@ See [stylelint options](http://stylelint.io/user-guide/node-api/#options) for th
 * `lintDirtyModulesOnly`: Lint only changed files, skip lint on start. Default: `false`
 * [`syntax`](https://stylelint.io/user-guide/node-api/#syntax): e.g. use `'scss'` to lint .scss files. Default: `undefined`
 
+## Use With Webpack Reporters
+
+When using this plugin with a webpack reporter, such as
+[webpack-stylish](https://github.com/webpack-contrib/webpack-stylish), you must
+set the `formatter` option to `'webpack'` for proper error and warning output. eg:
+
+```js
+plugins: [
+  new StyleLintPlugin({
+    formatter: 'webpack'
```

Comment:
I would prefer explicitly `require`ing the function, i.e. `formatter: require('stylelint-webpack-plugin/lib/webpack-formatter')` to keep the signature consistent and remove an (IMO) unnecessary branch in `index.js`. What do you think?

Reply:
Experience on the core team points to that being another unnecessary step for users (that's been a very big public gripe recently)